core/src/main/scala/org/http4s/QueryOps.scala

```diff
@@ -36,20 +36,24 @@ trait QueryOps {
     _withQueryParam(QueryParam[T].key, values.map(QueryParamEncoder[T].encode))
 
   /** alias for withQueryParam */
-  def +?[K: QueryParamKeyLike, T: QueryParamEncoder](name: K, value: T): Self =
-    +?(name, value :: Nil)
+  def +?[K: QueryParamKeyLike, T: QueryParamEncoder](param: (K, T)): Self =
+    ++?((param._1, param._2 :: Nil))
 
   /** alias for withQueryParam */
   def +?[K: QueryParamKeyLike](name: K): Self =
     _withQueryParam(QueryParamKeyLike[K].getKey(name), Nil)
 
   /** alias for withQueryParam */
-  def +?[K: QueryParamKeyLike, T: QueryParamEncoder](name: K, values: collection.Seq[T]): Self =
-    _withQueryParam(QueryParamKeyLike[K].getKey(name), values.map(QueryParamEncoder[T].encode))
+  def ++?[K: QueryParamKeyLike, T: QueryParamEncoder](param: (K, collection.Seq[T])): Self =
+    _withQueryParam(
+      QueryParamKeyLike[K].getKey(param._1),
+      param._2.map((t: T) => QueryParamEncoder[T].encode(t)))
```

Comment:
This is a minutiae, but in my opinion it would be better to keep the same style between this method and the next one. I mean
```
param._2.map((t: T) => QueryParamEncoder[T].encode(t)))
```
versus
```
param._2.map(QueryParamEncoder[T].encode))
```

Reply:
Done! https://github.com/http4s/http4s/pull/3758/commits/f92cac396bace11dd523ca3540940c405d6aab0f